### ISSUE TITLE
Empty High Res & Low Res Folders

### ISSUE THREAD

**Issue by Callahan-Henry** [WPI-AIM/ambf_addon#26]
Hi!

I am currently trying to use the add on, but I am running into some issues.  I have tried to use both the blender-2.8x and blender-2.9x branches with their respective Blender versions and have run into the same issue.  When I save the meshes, I find that both the High Res and Low Res folders are empty.  When creating the ADF/yaml file, it is created, but only the bare bones of the file appear (there are no bodies or joints listed).  Due to both of these issues, the ambf simulation appears empty.  I have tried both adding the add on through preferences, as well as with the scripting option.  Neither of these options display any error messages.  I have also tried using different Blender models, but I get the same result.  I am new to Blender, so it is possible that I am forgetting to do something basic within Blender that is expected for this add on to work.  Thank you for your help in advance!

**Comment by adnanmunawar** [comment]
Hello Callahan,
Could you record a short video / GIF of how you are creating ADF files? I would also highly recommend using the Blender 3.x branch and the latest Blender version.

**Comment by Callahan-Henry** [comment]
I am currently using Blender 3.6.1 and the Blender 3.x branch.  In the following video, I show how I created the ADF file of the simple sphere.  As previously mentioned, the high and low res folders are created, but are empty, and the skinned_ball.yaml file is created, but is useless as it contains no information on the desired sphere.  Strangely, I have noticed that I can import an existing ADF file, edit in Blender, and re-create the meshes and ADF file just fine.  The problem seems to happen when importing an stl file into Blender, or simply using a mesh created in Blender in order to generate the ADF file.  Thanks again for the support! 

https://github.com/WPI-AIM/ambf_addon/assets/67883990/e06ee8a1-881c-4a30-b3d8-33ed4de99712
